Recipes/Recipes/Supporting Files/Fonts.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Fonts.swift
+//  Recipes
+//
+//  Created by Vitaliy Iakushev on 05.03.2024.
+//
+
+import UIKit
+
+/// LightWeight для Шрифтов
+extension UIFont {
```

Comment:
```suggestion
extension UIFont {
```
оформляем правильно экстеншен смотри как у других и в важном я давно писал